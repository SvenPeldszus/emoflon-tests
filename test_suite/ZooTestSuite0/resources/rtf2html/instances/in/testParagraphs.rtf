{\rtf1 {\pard 
A paragraph\par
\pard another one\par\pard 
It doesn't matter if we reset the paragraph sytle a priori, it still is a paragraph\par
}}