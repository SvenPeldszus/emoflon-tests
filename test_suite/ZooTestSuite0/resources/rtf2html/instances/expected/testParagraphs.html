<span><span style="rtfcommand: rtf1;"><span><p>
A paragraph</p>
<p>another one</p><p>
It doesn't matter if we reset the paragraph sytle a priori, it still is a paragraph</p>
</span></span></span>